test/integration/parachain/staging/system.test.ts

```diff
@@ -38,9 +38,16 @@ describe("systemAPI", () => {
 
     // TODO: Unskip once differences between rococo-local and standalone are fixed
     it.skip("should setCode", async () => {
-        const code = fs.readFileSync(
-            path.join(__dirname, "../../../mock/rococo_runtime.compact.wasm")
-        ).toString("hex");
+        const code = fs.readFileSync(path.join(__dirname, "../../../mock/rococo_runtime.compact.wasm")).toString("hex");
         await sudo(interBtcAPI, () => interBtcAPI.system.setCode(code));
     });
+
+    it("should getFutureBlockNumber", async () => {
+        const [currentBlockNumber, futureBlockNumber] = await Promise.all([
+            interBtcAPI.system.getCurrentBlockNumber(),
+            interBtcAPI.system.getFutureBlockNumber(120),
+        ]);
+
+        assert.isTrue(currentBlockNumber + 10 === futureBlockNumber);
```

Comment:
I wonder if we can run into some race condition / slippage there.

I'd suggest adding some wiggle room with a combined

```
assert.isAtLeast(futureBlockNumber, currentBlockNumber + 9);
assert.isAtMost(futureBlockNumber, currentBlockNumber + 11);
```

Otherwise, if we're certain it must be 10 add a meaningful error message in case it isn't.